<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="closebutton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 41, 41)</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="DataBase1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>dbc for can1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_dbc1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>browser</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_dbc1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="flag_OK1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="DataBase2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>dbc for can2</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_dbc2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_dbc2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>browser</string>
    </property>
   </widget>
   <widget class="QLabel" name="flag_OK2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>891</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>891</width>
        <height>271</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>891</width>
        <height>271</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closebutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_dbc1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button_action_dbc1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>686</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_dbc2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button_action_dbc2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Action()</slot>
  <slot>button_action_dbc1()</slot>
  <slot>button_action_dbc2()</slot>
 </slots>
</ui>
